content/blog/2020-06-29-june-20-community-gems.md

```diff
@@ -0,0 +1,164 @@
+---
+title: June '20 Community Gems
+date: 2020-06-29
+description: |
+  A roundup of technical Q&A's from the DVC community. This month, we discuss 
+  migrating to DVC 1.0, the new pipeline format, and our Python API.
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC community. This month, we discuss 
+  migrating to DVC 1.0, the new pipeline format, and our Python API.
+picture: 2020-06-29/Gems_June_20.png
+author: elle_obrien
+commentsUrl: https://discuss.dvc.org/t/june-20-community-gems/426
+tags:
+  - Discord
+  - Gems
+  - MinIO
+  - Pipeline
+  - Python API
+  - Optimization
+---
+
+## Highlights from Discord
+
+Here are some Q&A's from our Discord channel that we think are worth sharing.
+
+### Q: I just upgraded to DVC 1.0. I've got some pipeline stages currently saved as `.dvc` files. [Is there an easy way to convert the old `.dvc` format to the new `dvc.yaml` standard?](https://discord.com/channels/485586884165107732/563406153334128681/725019219930120232)
+
+Yes! You can easily transfer the stages by hand: `dvc.yaml` is designed for
+manual edits in any text editor, so you can type your old stages in and then
+delete the old `.dvc` files. We also have a
+[migration script](https://gist.github.com/skshetry/07a3e26e6b06783e1ad7a4b6db6479da)
+available, although we can't provide long-term support for it.
+
+Learn more about the `dvc.yaml` format in our
+[brand new docs](https://dvc.org/doc/user-guide/dvc-files-and-directories#dvcyaml-file)!
+
+https://media.giphy.com/media/JYpTAnhT0EI2Q/giphy.gif
+
+_Just like this but with technical documentation._
+
+### Q: After I pushed my local data to remote storage, I noticed the file names are different in my storage repository- they're hash values. [Can I make them more meaningful names?](https://discord.com/channels/485586884165107732/563406153334128681/717737163122540585)
+
+No, but for a good reason! What you're seeing are cached files, and they're
+stored in a special format that makes DVC versioning and addressing possible-
```

Comment:
format -> way (we don't change format, it might confuse some folks). CSV stays CSV, we only change its name

Reply:
good point

Reply:
i'm going to say "naming convention"

Reply:
This has been updated in the latest PR